app/models/card.rb

```diff
@@ -1,7 +1,23 @@
 class Card < ActiveRecord::Base
-  before_create :set_review
+  before_validation :set_review_date, on: :create
+  validate :original_translated_text_equal
+  validates :original_text, :translated_text, :review_date,
+            presence: { message: 'Необходимо заполнить поле.' }
 
-  def set_review
-    self.review = Time.now + 3.days
-  end
+  protected
+    def set_review_date
+      self.review_date = Time.now + 3.days
+    end
+
+    def original_translated_text_equal
+      self.original_text = full_downcase(original_text)
+      self.translated_text = full_downcase(translated_text)
+      if original_text == translated_text
+        errors.add(:original_text, 'Вводимые значения должны отличаться.')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.